doc/src/atom_style.rst

```diff
@@ -91,7 +91,7 @@ quantities.
 +--------------+-----------------------------------------------------+--------------------------------------+
 | *charge*     | charge                                              | atomic system with charges           |
 +--------------+-----------------------------------------------------+--------------------------------------+
-| *dielectric* | dipole, area, curvature                             | system with surface polarization     |
+| *dielectric* | normx normy normz area_per_patch ed em epsilon curv | system with surface polarization     |
```

Comment:
We typically don't like "_" (underscores) in the documentation, I would suggest changing `area_per_patch` to `area/patch`